Teleportation/Tests.qs

```diff
@@ -189,6 +189,28 @@ namespace Quantum.Kata.Teleportation {
         TeleportPreparedStateTestLoop(PrepareAndSendMessage_Reference, ReconstructAndMeasureMessage);
     }
     
+    @Test("QuantumSimulator")
+    operation T18_EntanglementSwapping () : Unit {
+        
+        let numRepetitions = 100;
```

Comment:
You're using this variable in just one place, so probably don't need a separate variable for it. (Also, if the loop starts with 0, the actual number of repetitions is one greater than the value of the variable)

Reply:
done